jedi_vim.py

```diff
@@ -417,9 +416,6 @@ def rename():
                 vim.current.window.cursor = r.start_pos
                 vim_command('normal! cw%s' % replace)
 
-            result = new_buffer(window_path)
-            if not result:
-                return
             vim.current.window.cursor = cursor
```

Comment:
Well the problem is, that I'm pretty sure you're breaking renaming over multiple files. So please include those lines again. Then we can talk about adding the other stuff.